### ISSUE TITLE
HMDI profile switching on LG TVs

### ISSUE THREAD

**Issue by L337Fool** [Maassoft/ColorControl#153]
I am attempting to have my LG TV switch from the HDMI PC profile to the plain HDMI profile whenever I start it then revert when I close the app. I realize I can do this manually through the expert options under remote control. Is this feature not available or am I missing something here?

**Comment by Maassoft** [comment]
Currently there is no way to automatically execute a LG profile on startup or when closing the app. You can however start presets from the command line:
`ColorControl.exe --lg-preset "Name of preset"`

Or you can just simply leave the app running and assign keyboard shortcuts to presets to start them without UI interaction.

**Comment by L337Fool** [comment]
> Currently there is no way to automatically execute a LG profile on startup or when closing the app. You can however start presets from the command line: `ColorControl.exe --lg-preset "Name of preset"`
> 
> Or you can just simply leave the app running and assign keyboard shortcuts to presets to start them without UI interaction.

Thanks, appreciate the fast response.

Is there a way to hotkey 

LG Controller>Expert>Misc>HDMI 1 Icon>HMDI 
then 
LG Controller>Expert>Misc>HDMI 1 Icon>PC 

As this would do the trick nicely?

**Comment by Maassoft** [comment]
Yes, you can create a new preset and then use the "Add step" button to add one or more actions (same ones as under the Expert button). To change the HDMI 1 icon you can use the menu path: "Add step -> Actions -> Misc -> HDMI 1 icon" and then choose the desired icon.
The end result for the steps would be:
`HDMI_1_icon(hdmigeneric), HDMI_1_icon(pc)`

**Comment by L337Fool** [comment]
Thank you so much for your support here and the time you've put into this!
